build/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
This file is already present at https://github.com/openebs/node-disk-manager/blob/master/build/custom-boilerplate.go.txt